### PR TITLE
Added text about which specification is authoritative.

### LINKED ISSUE #5: RTCStatsType out of sync with webrtc-pc
The [RTCStatsType in webrtc-stats](http://w3c.github.io/webrtc-stats/#idl-def-RTCStatsType) is out of sync with [RTCStatsType in webrtc-pc](https://w3c.github.io/webrtc-pc/#idl-def-RTCStatsType).

1. In webrtc-stats there is no hyphen before `rtp`.
2. There are missing stats types from the webrtc-pc documentation.

### PR DESCRIPTION
**Pull request by alvestrand**
Fixes #5
Partially addresses #55

### CODE REVIEW

File: webrtc-stats.html [hunk 1]
```diff
@@ -88,29 +88,55 @@
...
-      <code>RTCStats</code> represents.
-      <div class="note">
-        OPEN ISSUE: Need to define an IANA registry for the RTCStatsType and populate it with the
-        following set as baseline.
-      </div>
+      </h2>
+      <p>The <code>RTCStatsType</code> member indicates the type of the
+      object that the
+      <code>RTCStats</code> object represents. An object with a given "type"
+      can be only one type of dictionary, but multiple "type" values may use
```
Comment (vr000m) [inline webrtc-stats.html]: missing: use

can be only /use/ one type of dictionary

File: webrtc-stats.html [hunk 2]
```diff
@@ -88,29 +88,55 @@
         [[!WEBRTC]].
       </p>
     </section><!-- VS: getStats() remains in base spec! -->
+    <section class="informative">
+      <h2>Basic concepts</h2>
+      <p>The stats API is defined in [[!GETUSERMEDIA]]. It is defined to return
```
Comment (alvestrand) [inline webrtc-stats.html]: Oops. Yes.

File: webrtc-stats.html (hunk 1 above)
Comment (alvestrand) [inline webrtc-stats.html]: I think it's grammatically correct as it stands, but hard to read. Reformulating.

**Comment by alvestrand** [comment]
@vr000m please take another look

**Pull request merged**